subsys/bluetooth/audio/bap_stream.c

```diff
@@ -781,7 +781,7 @@ int bt_bap_stream_release(struct bt_bap_stream *stream)
 	LOG_DBG("stream %p", stream);
 
 	CHECKIF(stream == NULL || stream->ep == NULL || stream->conn == NULL) {
-		LOG_DBG("Invalid stream");
+		LOG_DBG("Invalid stream (ep %p, conn %p)", stream->ep, (void *)stream->conn);
```

Comment:
```suggestion
		LOG_DBG("Invalid stream (ep %p, conn %p)", stream->ep, stream->conn);
```

Reply:
The `(void *)` cast is necessary for the logging system, when logging pointers to opaque structs (like `bt_conn`) where the source file does not know the internals. 

I don't know why or if it can be fixed in a general way, but it's something that we need to do right now (and have done several places).